repository/src/main/resources/db/useful-queries/Useful select queries.sql

```diff
@@ -0,0 +1,95 @@
+-- ==========================================
+-- Details about tables within ccd-data-store
+-- ==========================================
+-- PK/FK relationships in tables case_data/case_event
+-- case_data pk column = id
+-- case_event pk column = id, fk column = case_data_id
+-- case_event_significant_items, pk column = id
+-- case_users, pk combined columns = case_data_id + user_id + case_role
+-- all_events, no pk or fk relationships defined
+
+-- =================================================================================
+-- Queries below return data from the various tables within ccd-data-store
+-- The select queries are ordered such that they can be used to delete ALL data by a
+-- given Jurisdiction. 
+-- I.E to perform a delete replace 'SELECT *' with 'DELETE'. Replace 'EMPLOYMENT' 
+-- with your Jurisdiction.
```

Comment:
Better to use a placeholder jurisdiction name instead of EMPLOYMENT, to reduce chances of a mistaken deletion in a work environment.